src/frontend/Komponenter/Behandling/Brevmottakere/BrevmottakereModal.tsx

```diff
@@ -90,12 +89,32 @@ export const BrevmottakereModal: FC<{
         valgteIdenter.some((identNummer) => !initelleIdenter.includes(identNummer)) ||
         valgteIdenter.length !== initelleIdenter.length;
 
+    const harNyeOrganisasjoner = (
+        initielleOrgNumre: IOrganisasjonMottaker[],
+        valgteOrgNumre: IOrganisasjonMottaker[]
+    ) => {
+        if (initielleOrgNumre.length !== valgteOrgNumre.length) {
+            return true;
+        }
+        const initielleOrganisasjonsNummer = initielleOrgNumre.map(
+            (organisasjonsMottaker) => organisasjonsMottaker.organisasjonsnummer
+        );
+        const initielleNavHosOrganisasjon = initielleOrgNumre.map(
+            (organisasjonsMottaker) => organisasjonsMottaker.navnHosOrganisasjon
+        );
+        return valgteOrgNumre.some(
+            (organisasjonsMottaker) =>
+                !initielleOrganisasjonsNummer.includes(organisasjonsMottaker.organisasjonsnummer) ||
+                !initielleNavHosOrganisasjon.includes(organisasjonsMottaker.navnHosOrganisasjon)
+        );
+    };
+
     const harBrevmottakere =
         valgtePersonMottakere.length > 0 || valgteOrganisasjonMottakere.length > 0;
 
     const mottakerListeHarEndringer =
         harNyeMottakere(initiellePersonIdenter, valgtePersonIdenter) ||
-        harNyeMottakere(initelleOrgNumre, valgteOrgNumre);
+        harNyeOrganisasjoner(initielleOrgNumre, valgteOrganisasjonMottakere);
 
     const deaktiverSettMottakere = !harBrevmottakere || !mottakerListeHarEndringer;
```

Comment:
Kunne denne || !mottakerListeHarEndringer; slettes? 
Har ikke lyst til å teste det uten å teste nøye, men det ville blitt mindre kode!

Reply:
Ja, at vi alltid kan sette nye mottakere uten noen form for sjekk om det er unødvendig å sette mottakere på nytt ?